port/unix/omrmmap.c

```diff
@@ -318,7 +433,8 @@ omrmmap_capabilities(struct OMRPortLibrary *portLibrary)
   || (defined(LINUX) && defined(AARCH64)) \
   || (defined(LINUX) && defined(RISCV64)) \
   || (defined(AIXPPC)) \
-  || (defined(OSX)))
+  || (defined(OSX)) \
+  || (defined(J9ZOS390)))
```

Comment:
If you inserted J9ZOS390 before OSX (where it fits alphabetically) you wouldn't need to change a second line here.

Reply:
Fixed.